css/styles.css

```diff
@@ -479,9 +479,18 @@ input[type=checkbox]:checked:after {
     display: flex;
     align-items: center;
     flex-direction: column;
-    padding: 2rem 1.5rem;
-    border-radius:17px;
-    box-shadow:5px 5px 20px black;
+    padding: 1.7rem 1.6rem;
+    background: rgba(0, 0, 0, 0.07);
+    box-shadow: 0 5px 5px rgba(0, 0, 0, 0.5);
+    border-radius: 20px;
+    transition: all 265ms ease-out;
+}
+
+.member:hover {
+    padding: 1.9rem 1.8rem;
+    margin-bottom:20px;
+    background: rgba(0, 0, 0, 0.05);
```

Comment:
Changing the padding and margin to animate the cards causes the text to move on the page, I would suggest using transform instead. Also I think the card should get darker instead of lighter on hover to contrast more.

Try this one and let me know what you think:
```css
.member:hover {
    transform: translateY(-5px) scale(1.05);
    background: rgba(0, 0, 0, 0.15);
    transition: all 265ms ease-out;
```

Reply:
I agree with you, I think that it looks better with the using transform. Thank you for your suggest!